pyqir/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [project]
-name = "test-pyqir"
+name = "iqm-pyqir"
 version = "0.10.7"
```

Comment:
Would it make sense to add `DEVELOPMENT.md` file and mention that version number has to be incremented manually for every release?

Also, it might make sense to switch versioning from "semantic versioning" format (MAJOR.MINOR.PATCH) to "compatibility versioning" format (MAJOR.MINOR).

We could also add a `CHANGELOG.md` file.